backpack-common/src/main/java/net/skyscanner/backpack/calendar2/CalendarParams.kt

```diff
@@ -145,6 +145,19 @@ data class CellInfo(
     }
 }
 
+/**
+ * Describes the label to be added to the cell
+ *
+ * @param text the text to be displayed
+ * @param icon the icon to be displayed
+ * @param iconTint the tint of the icon
+ */
+data class CellLabel(
+    val text: String? = null,
+    val icon: Int? = null,
```

Comment:
Is there a reason for not using `BpkIcon` here? In our case we could pass `BpkIcon.Search` in. Looking at the screenshots in this PR, the icon is visibly pixelated due to using the smaller version. The size adaptability we'd get out of the box with `BpkIcon`.

Reply:
@andra-georgescu To use BpkIcon directly, I would need to add the compose module as a dependency in the view-based module. Since this API is currently shared between both the compose and view-based versions of the calendar, this solution was the simplest one that came to mind. I'm open to exploring other options as well.